cmd/prometheus/main.go

```diff
@@ -259,6 +262,31 @@ func reloadConfig(filename string, rls ...Reloadable) (err error) {
 		return fmt.Errorf("couldn't load configuration (-config.file=%s): %v", filename, err)
 	}
 
+	// Add AlertmanagerConfigs for legacy Alertmanager URL flags.
```

Comment:
You mean the deprecated way and the one that doesn't work yet. I'm fairly sure this will work :)